build/all/Dockerfile

```diff
@@ -55,6 +55,7 @@ ARG USER_GID=$USER_UID
 RUN groupadd --gid $USER_GID $USERNAME && \
   useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
 USER nonroot:nonroot
+RUN apt-get update && apt-get install -y bash git
```

Comment:
Does doing the update here actually avoid CVEs? Or would using the latest DEBIAN_BASE_IMAGE already solve that problem for packages that aren't installed by default?

Reply:
I don't think the base image is guaranteed to have an up to date package index. It seems plausible that packages which aren't already installed may be indexed to a stale version which has since been patched. I don't think the upstream base image would be updated for packages which are not installed, thus I don't think we can rely on just updating to the latest base image